### PR TITLE
GA for MIP-MCAS integration

### PR DESCRIPTION
**Pull request by idbasre**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/dlp-use-policies-non-microsoft-cloud-apps.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ Before you start using DLP policies to non-Microsoft cloud apps, confirm your [M
 - Microsoft 365 E5 Compliance
 - Microsoft 365 E5 Security
 
+### Permissions
+The user who creates the DLP policy should be:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-use-policies-non-microsoft-cloud-apps.md]:
```suggestion
The user who creates the DLP policy should be a:
```

File: microsoft-365/compliance/dlp-use-policies-non-microsoft-cloud-apps.md [hunk 2]
```diff
@@ -76,7 +82,7 @@ When you create a rule in the DLP policy, you can select an action for non-Micro
 
 ![Restrict third-party apps](../media/4-dlp-non-microsoft-cloud-app-restrict-third-party-apps.png)
 
->[NOTE]
+>[!NOTE]
 >DLP policies applied to non-Microsoft apps use Microsoft Cloud App Security. When the DLP policy for a non-Microsoft app is created, the same policy will be automatically created in Microsoft Cloud App Security.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-use-policies-non-microsoft-cloud-apps.md]:
```suggestion
> [!NOTE]
> DLP policies applied to non-Microsoft apps use Microsoft Cloud App Security. When the DLP policy for a non-Microsoft app is created, the same policy will be automatically created in Microsoft Cloud App Security.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @idbasre, kindly commit the few minor suggested changes. Many thanks.

**Comment by idbasre** [comment]
@JohanFreelancer9 I have committed the changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by idbasre** [comment]
@chrfox Hi, there is an ETA when it will be published?

**Pull request merged**